src/Samples/DIPS.Xamarin.UI.Samples/Controls/Sheet/SheetPage.xaml

```diff
@@ -15,42 +15,51 @@
-                                    VerticalContentAlignment="{Binding VerticalContentAlignment}"
-                                    ContentColor="{Binding ContentColor}"
-                                    HeaderColor="{Binding HeaderColor}"
-                                    HandleColor="{Binding HandleColor}"
-                                    HasShadow="{Binding HasShadow}"
-                                    Position="{Binding Position}"
-                                    MinPosition="{Binding MinPosition}"
-                                    MaxPosition="{Binding MaxPosition}"
-                                    BindingContextFactory="{Binding SheetViewModelFactory}"
-                                    CloseOnOverlayTapped="{Binding Source={x:Reference ShouldCloseOnOverlayTappedCheckbox}, Path=IsChecked}"
-                                    ShouldRememberPosition="{Binding ShouldRememberPosition}">
-                    <StackLayout>
-                        <Label Text="{Binding Source={x:Reference entry}, Path=Text}" />
-                        <Label Text="{Binding Source={x:Reference entry}, Path=Text}" />
-                        <Label Text="{Binding Source={x:Reference entry}, Path=Text}" />
-                        <Label Text="{Binding Source={x:Reference entry}, Path=Text}" />
-                        <Label Text="{Binding Source={x:Reference entry}, Path=Text}" />
-                        <Entry x:Name="entry"
-                               Placeholder="TextEntry" />
-                        <Button Text="Close sheet"
-                                dxui:Modality.CloseOnClick="True" />
-                    </StackLayout>
-                </dxui:SheetBehavior>
-            </dxui:ModalityLayout.Behaviors>
-
+        <dxui:ModalityLayout.Behaviors>
+            <dxui:SheetBehavior IsOpen="{Binding IsSheetOpen}"
+                                OnOpenCommand="{Binding OnOpenCommand}"
+                                OnOpenCommandParameter="Opened"
+                                OnCloseCommand="{Binding OnCloseCommand}"
+                                OnCloseCommandParameter="Closed"
+                                IsDraggable="{Binding IsDraggable}"
+                                Alignment="{Binding Alignment}"
+                                VerticalContentAlignment="{Binding VerticalContentAlignment}"
+                                ContentColor="{Binding ContentColor}"
+                                HeaderColor="{Binding HeaderColor}"
+                                HandleColor="{Binding HandleColor}"
+                                HasShadow="{Binding HasShadow}"
+                                Position="{Binding Position}"
+                                MinPosition="{Binding MinPosition}"
+                                MaxPosition="{Binding MaxPosition}"
+                                ActionCommand="{Binding ActionCommand}"
+                                ActionCommandParameter="Hei"

```

Comment:
No need to add these parameters, it's obvious that we have parameters 👍 You will probably have to change the commands in the code behind view model as well.